velox/vector/DecodedVector.h

```diff
@@ -200,6 +253,10 @@ class DecodedVector {
   /// have been previously decoded by 'this'. This is used when 'data'
   /// is a component of the base vector of 'wrapper' and must be used
   /// in the same context, thus with the same indirections.
+  /// NOTE: The nulls buffer returned may also have nulls from the base()
+  /// combined into it. To control which levels are combined, please make sure
+  /// to use makeIndices() instead of decoded() when initializing the
+  /// DecodedVector.
```

Comment:
Not sure if I understand this comment. Do you mean wrap() should only be used together with makeIndices with the max numLevels (that includes the base layer)? What would be a counterexample of a wrong usage with decoded()?

Reply:
I think this note is unnecessarily causing confusion so i removed it. What I meant here was kind of a warning that decode() merges all layers and if you only want to merge layers upto a certain level then use makeIndices() instead.